src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -222,15 +232,82 @@ protected void runCommand(ExecutorDriver driver, Task task, String command) {
     }
   }
 
+  protected boolean unhealthy(ExecutorDriver driver, Task task) {
```

Comment:
Can we name this method checkTaskHealth. I think that is a more descriptive name than unhealthy.